src/applications/mhv/medical-records/sass/record-list.scss

```diff
@@ -29,6 +32,11 @@
     border-bottom: 2px solid $color-black !important;
     border-radius: 0px;
     background-color: $color-white !important;
+    max-width: none;
+    padding: 16px 0 24px;
+    h3 {
+      margin-bottom: 16px;
+    }
```

Comment:
Do we have to use px values here for the PDFs or can we use the design system classes?